app/assets/stylesheets/variables.scss

```diff
@@ -1,3 +1,4 @@
 $primary: #1A454A;
+$primary-light: #{$primary} + '33';
```

Comment:
I've tried different options, and this is by far the best for what we are trying to do. Tested with many HEX values, and it dims the colour as expected. Lighten changes the colour too much.